### PR TITLE
Improve customize registration form documentation page

### LINKED ISSUE #142: Overriding registration form
## Is it possible to override registration form?

I need to have users to accept the terms during registration. This doesn't come with default registration form. Is it possible to override the form?

### PR DESCRIPTION
**Pull request by Bartololo12**
Closes #142 

## Improved documentation page for customizing registration form

Documentation changed to provide more details on how to customize registration for in NucleosProfileBundle

### CODE REVIEW

**Comment by Bartololo12** [comment]
@core23 I am not sure why the docs checks are failing. I verified the docs with Online Sphinx editor and it looks ok. Do you have any advice how can I verify what is wrong with my commits?

File: docs/customize_registration.rst [hunk 1]
```diff
@@ -5,9 +5,75 @@ Customize Registration
...
+                model: App\Form\Model\Registration
+
+3. Use Symfony Form Extensions to add fields. You can use builder to remove or modify fields as well.
+
+.. code-block:: php-annotations
+
+    namespace App\Form\Type;
+
+    use Symfony\Component\Form\AbstractTypeExtension;
+    use Symfony\Component\Form\FormBuilderInterface;
```
Comment (core23) [inline docs/customize_registration.rst:64]: You need to reorder this imports @Bartololo12

Comment (Bartololo12) [reply docs/customize_registration.rst]: Thanks, that helped for DoctorRST, build is still failing.

**Comment by core23** [comment]
There was an error in the workflow. Thanks @Bartololo12

**Comment by alexsegura** [comment]
@core23 can you publish it on docs.nucleos.rocks please? It's still showing the previous version.

**Comment by core23** [comment]
I'll have a look at this. Thanks

**Pull request merged**